compiler/src/dotty/tools/dotc/printing/RefinedPrinter.scala

```diff
@@ -513,7 +513,7 @@ class RefinedPrinter(_ctx: Context) extends PlainPrinter(_ctx) {
           keywordStr("throw ") ~ toText(expr)
         }
       case SeqLiteral(elems, elemtpt) =>
-        "[" ~ toTextGlobal(elems, ",") ~ " : " ~ toText(elemtpt) ~ "]"
```

Comment:
A SeqLiteral is a specific kind of tree node, when debugging the compiler output with -Xprint, it's useful to be able to distinguish a SeqLiteral produced by the compiler from a Seq tree written by the user. If this doesn't appear in something that the compiler prints to the user then I don't think it should be changed.

Reply:
Agree with @smarter. We should at least retain the old output under -Yprint-debug